sdk/core/loader/types.h

```diff
@@ -492,6 +501,8 @@ namespace loader
 				Scheduler,
 				/// The memory allocator
 				Allocator,
+				/// The token server
+				TokenServer,
```

Comment:
This probably wants to be consistently named with the accessor.  I'd suggest `AllocatorLibrary`.